resources/migrations/049_add_pulse_skip_if_empty.yaml

```diff
@@ -0,0 +1,15 @@
+databaseChangeLog:
+  - changeSet:
+      id: 49
```

Comment:
we already have a migration 49 and 50 should be merged shortly so this should be `051_add_pulse_skip_if_empty.yaml`